values.schema.json.tpl

```diff
@@ -226,6 +226,20 @@
                         }
                     }
                 },
+                "podLabels": {
+                    "description": "Adds labels to every pod created",
+                    "type": [
+                        "object",
+                        "null"
+                    ]
+                },
+                "svcLabels": {
+                    "description": "Adds labels to every service created",
+                    "type": [
+                        "object",
+                        "null"
+                    ]
+                },
```

Comment:
The schema for both `podLabels` and `svcLabels` should use "properties" like `resourceLabels` so that they can be input correctly on the Openshift UI.

Reply:
Done!